doc/RecordingControl.xml

```diff
@@ -2039,13 +2041,103 @@ secfrac = "." 1*6DIGIT</programlisting>
       </para>
       <para>
+        When using <literal>StaticKey</literal> then the device shall interpret the configured <literal>KID</literal> for the encryption entry as a 16-byte hexadecimal value and use this value as the key identifier.
+        The device shall include a <xref linkend="_refKeyStorageStandardCenc" /> Standard CENC PSSH box.
+      </para>
+      <para>
+        When using <literal>Certificate</literal> then the device shall generate a <literal>KID/Key</literal> pair and encrypt the <literal>Key</literal> with each certificate public key defined in the configuration. The encrypted keys shall be stored in the file using a <xref linkend="_refKeyStorageKeyRotationSystem" /> Key rotation system PSSH box.
+        When <literal>KeyRotationDuration</literal> is set then the device shall generate a new <literal>KID/Key</literal> pair at the specified time interval as defined by <literal>KeyRotationDuration</literal>.
+        New segments shall use the latest generated <literal>Key</literal> for its encryption.
+      </para>
+      <para>
+        The device shall create a unique initialization vector for each segment.
+        Each encrypted file shall include the <literal>moov</literal> box containing the required PSSH box(es) according to the encryption entry configuration.
         If an encryption entry is reconfigured for an active recording, the device shall continue to use the old configuration for any open segments and shall start to use the new configuration for new segments.
       </para>
+      <section>
+        <title>Encryption keys storage</title>
+        <para>
+          When frame encryption is configured, the device shall store relevant information about the encryption keys used in the files with 'Protection system specific header' (or 'pssh') boxes [ISO/IEC 23001-7].
+        </para>
+        <section xml:id="_refKeyStorageStandardCenc">
+          <title>Standard CENC system</title>
+          <para>
+            When a file is frame encrypted exactly one pssh box of type 'CENC' box shall be present in the file according to [W3C 'cenc' Initialization Data Format].
+            This is independent of the <literal>EncryptionMode</literal>.
+            Additionnal types of 'pssh' boxes may be present to support alternative DRM systems, such as <xref linkend="_refKeyStorageKeyRotationSystem" />.
+          </para>
+          <section>
+            <title>Syntax</title>
+          <programlisting><![CDATA[
+aligned(8) class ProtectionSystemSpecificHeaderBox extends FullBox('pssh', version=1, flags=0)
+{
+  unsigned int(8)[16] SystemID;
+  unsigned int(32) KID_count;
+  for (i=1; i <= KID_count; i++){
+    unsigned int(8)[16] KID;
+  }
+  unsigned int(32) DataSize = 0;
+}
+            ]]></programlisting>
+            <para>See <a href="https://www.w3.org/TR/eme-initdata-cenc/#example">example</a> of a CENC pssh box by W3C.</para>
```

Comment:
The reference should point to the normative section instead of the example.
The reference needs to be provided as xlink to conform to docbook.

Reply:
done